From e8d7f5573998bef1dc5e0acb776694e4f262c17b Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Wed, 31 May 2023 14:16:55 +0200
Subject: [PATCH 0389/1141] arm64: dts: st: add PCIe PME interrupts on
 stm32mp251

PME MSI interrupts are handled by the GIC.
Provide here the irq number to be used by the portdrv

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I0680d6b6b1cf2c626e035fa0141d1a1612241c39
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/309763
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -1980,8 +1980,9 @@
 						<0 0 0 2 &intc 0 0 GIC_SPI 265 IRQ_TYPE_LEVEL_HIGH>,
 						<0 0 0 3 &intc 0 0 GIC_SPI 266 IRQ_TYPE_LEVEL_HIGH>,
 						<0 0 0 4 &intc 0 0 GIC_SPI 267 IRQ_TYPE_LEVEL_HIGH>;
-				interrupts = <GIC_SPI 232 IRQ_TYPE_LEVEL_HIGH>;
-				interrupt-names = "aer_msi";
+				interrupts = <GIC_SPI 232 IRQ_TYPE_LEVEL_HIGH>,
+					     <GIC_SPI 238 IRQ_TYPE_LEVEL_HIGH>;
+				interrupt-names = "aer_msi", "pme_msi";
 				#address-cells = <3>;
 				#size-cells = <2>;
 				ranges = <0x01000000 0 0x10010000 0x10010000 0 0x10000>,
